handbook/engineering/distribution/goals.md

```diff
@@ -2,41 +2,83 @@
 
 Goals are continuously updated and reviewed. If you find these goals do not reflect our current priorities or are out of date, please update them as soon as possible or add it as a topic to our [weekly sync](recurring_processes.md#weekly-distribution-team-sync).
 
-## Medium-term goals
+## Medium-term goals (~3-6 Months)
 
 ### Any engineer at Sourcegraph can create a release for all of our supported deployment types by running a single command
 
 Creating a new release for our deployments is currently a semi-automated process, which requires several manual steps and synchronizing our versioned artifacts (Sourcegraph, Kubernetes manifests, docker-compose manifests, etc). We want to enable any engineer to perform a release as often as needed, to enable this we want to make releasing Sourcegraph a simple, automated process.
 
-- **Owner**: Distribution Team
+- **Owner**: Stephen, Dave, Robert
 - **Status**: In Progress
+- **Length**: ~3 months
```

Comment:
I would like to move away from relative times because it is unclear what they are relative to.

```suggestion
- **Status**: In progress. Estimated completion by end of 2020.
```